src/twister2/yaml_test_specification.py

```diff
@@ -55,3 +64,111 @@ def __post_init__(self):
         self.extra_args = string_to_list(self.extra_args)
         self.integration_platforms = string_to_list(self.integration_platforms)
         self.timeout *= self.timeout_multiplier
+
+    @property
+    def scenario(self):
+        return self.build_name or self.original_name
+
+    @property
+    def build_dir(self) -> Path:
+        return (
+            self.output_dir / self.platform / self.rel_to_base_path / self.scenario
+        )
+
+
+# Using marshmallow to validate specification from yaml
+class HarnessConfigSchema(Schema):
+    type = fields.Str()
+    fixture = fields.Str()
+    ordered = fields.Str()
```

Comment:
should be Bool?
'''
E   marshmallow.exceptions.ValidationError: {'tests': defaultdict(<class 'dict'>, {'kernel.logging.message_capture': {'value': {'harness_config': {'ordered': ['Not a valid string.']}}}})
'''

Reply:
Fixed on main branch